src/ResourceManager/Websites/Commands.Websites/help/AzureRM.Websites.md

```diff
@@ -87,28 +87,27 @@ Removes an Azure App Service plan.
 Removes an Azure Web App.
 
 ### [Remove-AzureRmWebAppBackup](Remove-AzureRmWebAppBackup.md)
-Removes an Azure Web App backup.
+
 
 ### [Remove-AzureRmWebAppSlot](Remove-AzureRmWebAppSlot.md)
-Removes an Azure Web App slot.
+
 
 ### [Remove-AzureRmWebAppSSLBinding](Remove-AzureRmWebAppSSLBinding.md)
 Removes an SSL binding from an uploaded certificate.
 
 ### [Reset-AzureRmWebAppPublishingProfile](Reset-AzureRmWebAppPublishingProfile.md)
-Resets the publishing credentials of an Azure Web App.
+
 
 ### [Reset-AzureRmWebAppSlotPublishingProfile](Reset-AzureRmWebAppSlotPublishingProfile.md)
-Resets the publishing credentials of an Azure Web App slot.
+
 
 ### [Restart-AzureRmWebApp](Restart-AzureRmWebApp.md)
 Restarts an Azure Web App.
 
 ### [Restart-AzureRmWebAppSlot](Restart-AzureRmWebAppSlot.md)
-Restarts an Azure Web App slot.
+
 
 ### [Restore-AzureRmWebAppBackup](Restore-AzureRmWebAppBackup.md)
-Restores an Azure Web App from a backup.
```

Comment:
Do not remove all of these cmdlet descriptions. The help should be regenerated as this file is missing the entry for the new cmdlets in this PR.